CMakeLists.txt

```diff
@@ -1,7 +1,9 @@
+
 cmake_minimum_required(VERSION 2.8.6)
 
 ## The path to the FindEigen file.
 list(APPEND CMAKE_MODULE_PATH "${PROJECT_SOURCE_DIR}/cmake")
+set(CMAKE_CXX_FLAGS_RELEASE "${CMAKE_CXX_FLAGS_RELEASE} -O3")
```

Comment:
Indeed, you're right. I think I added it out of paranoia because CGAL was
monkeying with the default cmake flags.

On Mon, Mar 16, 2015 at 2:27 PM, Yotam Gingold <notifications@github.com>
wrote:

> In CMakeLists.txt
> <https://github.com/yig/SubdivisionSkinning/pull/1#discussion_r26513495>:
>
> >  cmake_minimum_required(VERSION 2.8.6)
> >
> >  ## The path to the FindEigen file.
> >  list(APPEND CMAKE_MODULE_PATH "${PROJECT_SOURCE_DIR}/cmake")
> > +set(CMAKE_CXX_FLAGS_RELEASE "${CMAKE_CXX_FLAGS_RELEASE} -O3")
>
> I'm pretty sure this line isn't necessary. Try creating an empty directory
> and putting a CMakeLists.txt in it containing:
>
> cmake_minimum_required(VERSION 2.8.6)
>
> message("${CMAKE_CXX_FLAGS_RELEASE}")
>
> and then running "cmake ."
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/yig/SubdivisionSkinning/pull/1/files#r26513495>.
>